### ISSUE TITLE
SELU activation in TFlite micro

### ISSUE THREAD

**Issue by opprud** [tensorflow/tensorflow#47424]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): **2.5**
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.** Currently no good alternative exists for deployment if micro. Batch Norm for micros seem to have som issues in certain Cortex M4 platforms.

**Will this change the current api? How?** It will add SELU activation to the supported ops in TF Lite Micro

**Who will benefit with this feature?** Embedded developers, interested in building NN's with multple layers.

**Any Other info.**
SELU activation as per [Klaumbauer et.al](https://arxiv.org/abs/1706.02515) provides self normalising, similar to Batch Normalisation, but requiring less compute, as it is an activation function. 
At the time of writing we haven't found alternatives, to running self normalising on microprocessor implementations. 

The application on wich the operator is desired is a predicitive maintainence system we are developing.
Our model is trained on field data captured from a large number of rotating machines, where the data mean and variance differs a lot, from from machine to machine, caused by mechanical tolerances and different mechanical transfer functions. Thus a degree of normalisation through the network is required, if we are to use identical models for detecting emerging defect in rotating components. 
We are using a convolutional model, fed with raw data, to be able to capture transient events, as opposed to traditional FFT based feature extraction, based on [this proposal](https://www.researchgate.net/publication/314247372_A_New_Deep_Learning_Model_for_Fault_Diagnosis_with_Good_Anti-Noise_and_Domain_Adaptation_Ability_on_Raw_Vibration_Signals) 

From what i can see the SELU could be implemented similarily to the ELU `/lite/kernels/elu.cc` that uses a lookuptable when performing inference, and the selu itself, works as sketched below  :

```
#define ALPHA   1.6733f
#define LAMBDA  1.0507f

inline void Selu(const RuntimeShape& input_shape, const float* input_data,
                const RuntimeShape& output_shape, float* output_data) {
  const int flat_size = MatchingFlatSize(input_shape, output_shape);
  for (int i = 0; i < flat_size; ++i) {
    const float val = input_data[i];
    output_data[i] = val < 0.0f ? LAMBDA * (ALPHA * std::expm1(val) - ALPHA) : LAMBDA * val;
  }
}
```

**Comment by lekshmysudha** [comment]
Hi, are there any updates on this? I have a DNN with SeLu activation for the hidden layers. I am trying to deploy the DNN on an ARM Cortex M4 MCU. I still can't see SeLu in the allowed Op list. Can using SELECT_TF_OPS solve this in any way?

**Comment by pkgoogle** [comment]
Hi,

Thank you for opening this issue. Since this issue has been open for a long time, the code/debug information for this issue may not be relevant with the current state of the code base.

The TFLite team is constantly improving the framework by fixing bugs and adding new features. We suggest you try the latest TensorFlow [version](https://pypi.org/project/tensorflow/) with the latest compatible hardware configuration which could potentially resolve the issue. If you are still facing the issue, please create a new GitHub issue with your latest findings, with all the debugging information which could help us investigate.

Please follow the [release notes](https://github.com/tensorflow/tensorflow/blob/master/RELEASE.md) to stay up to date with the latest developments which are happening in the TFLite space.

Thanks.
